docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,207 @@
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllMagazine()
+    ```
+
+- Remove all recipes of Modification Attachment Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllModificationAttachment()
+    ```
+
+- Remove all recipes of Normal Attachment Category:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllNormalAttachment()
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByBullet:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByBullet(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByGrenade:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByGrenade(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByGun:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByGun(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByMagazine:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByMagazine(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByModificationAttachment:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByModificationAttachment(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeByNormalAttachment:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeByNormalAttachment(IIngredient)
+    ```
+
+- Removes Recipe that outputs given `ingredient` from given category. valid categories = "GUN", "ATTACHMENT_NORMAL", "ATTACHMENT_MODIFICATION", "BULLET", "MAGAZINE", "GRENADE":
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeInGroup(IIngredient, CraftingGroup)
+    ```
+
+- Removes recipe that outputs given `ingredient` from all categories:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.remove(IIngredient)
+    ```
+
+- groovyscript.wiki.mwc.crafting_station.removeAllInGroup:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAllInGroup(CraftingGroup)
+    ```
+
+- Removes all registered recipes:
+
+    ```groovy:no-line-numbers
+    mods.mwc.crafting_station.removeAll()
+    ```
+
+:::::::::: details Example {open id="example"}
+```groovy:no-line-numbers
+mods.mwc.crafting_station.removeAllBullet()
+mods.mwc.crafting_station.removeAllGrenade()
+mods.mwc.crafting_station.removeAllGun()
+mods.mwc.crafting_station.removeAllMagazine()
+mods.mwc.crafting_station.removeAllModificationAttachment()
+mods.mwc.crafting_station.removeAllNormalAttachment()
+mods.mwc.crafting_station.removeByBullet(ore('oreDiamond'))
+mods.mwc.crafting_station.removeByGrenade(ore('oreDiamond'))
+mods.mwc.crafting_station.removeByGun(ore('oreDiamond'))
+mods.mwc.crafting_station.removeByMagazine(ore('oreDiamond'))
+mods.mwc.crafting_station.removeByModificationAttachment(ore('oreDiamond'))
+mods.mwc.crafting_station.removeByNormalAttachment(ore('oreDiamond'))
+mods.mwc.crafting_station.remove(ore('oreDiamond'))
```

Comment:
`remove`? remove by what? could be input, could be output. could do both in the same thing. make the method name descriptive.